packages/eslint-plugin/docs/rules/component-class-suffix.md

```diff
@@ -0,0 +1,80 @@
+# Enforces components to have a specified suffix (`component-class-suffix`)
+
+This rule enforces that classes decorated with @Component must have suffix "Component" (or custom) in their name.
+
+## Rule Details
+
+This rule follows the recommendation from the [Angular styleguide](https://angular.io/guide/styleguide#style-02-03).
+
+Examples of **incorrect** code for this rule (with default configuration):
+
+```javascript
```

Comment:
These should all be ` ```ts `, rather than ` ```javascript`

Reply:
Oh, I completely missed it haha, thanks!